eventsourcing/src/main/java/org/axonframework/eventsourcing/EventSourcingRepository.java

```diff
@@ -298,6 +302,13 @@ public Builder<T> spanFactory(SpanFactory spanFactory) {
             return this;
         }
 
+        @Override
```

Comment:
Does the `@Deprecated` method bubble up through an override? If not, I think we should also deprecate the override of the `spanFactory(SpanFactory)` method.